Makefile

```diff
@@ -106,9 +107,8 @@ endif
 	echo "set initrd elemental-${FINAL_TAG}-initrd" >> build/elemental-${FINAL_TAG}.ipxe
 	echo "set rootfs elemental-${FINAL_TAG}.squashfs" >> build/elemental-${FINAL_TAG}.ipxe
 	echo "set iso    elemental-${FINAL_TAG}.iso" >> build/elemental-${FINAL_TAG}.ipxe  #not used anymore, check if we can boot from iso directly with sanboot?
-	echo "kernel ${url}/${kernel} initrd=${initrd} ip=dhcp rd.cos.disable root=live:${url}/${rootfs} console=tty1 console=ttyS0 ${cmdline}"  >> build/elemental-${FINAL_TAG}.ipxe
-	echo "initrd ${url}${initrd}"  >> build/elemental-${FINAL_TAG}.ipxe
-	echo "boot" >> build/elemental-${FINAL_TAG}.ipxe
+	echo "initrd \$${url}/\$${initrd}"  >> build/elemental-${FINAL_TAG}.ipxe
```

Comment:
Any reason for the double $$ and the escape char? This should end up as ${url} in the file no? This is a simple echo no?

I probably mangled it as I have no idea about ipxe files or format

Reply:
Yes we need to have `${url}` in the ipxe file, it's a variable in the script (like in bash). So current version doesn't work because of this, for example we have `kernel / initr= ip=dhcp [...]` instead of `kernel ${url}/${kernel} initrd=${initrd} ip=dhcp [...]`.